pkg/util/gce/gce.go

```diff
@@ -88,6 +88,10 @@ func getResponse(url string) (string, error) {
 		return "", fmt.Errorf("GCE hostname, error reading response body: %s", err)
```

Comment:
Just noticed that this line uses "GCE hostname" in its message while its placed in the generic `getResponse` func. Should we perhaps remove this?